pkgs/misc/lilypond/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
         # Override default argv[0] setting so LilyPond can find
         # its Scheme libraries.
         wrapProgram "$f" --set GUILE_AUTO_COMPILE 0 \
-                         --set PATH "${ghostscript}/bin" \
+                         --set PATH "${ghostscript}/bin:${coreutils}/bin:$out/bin" \
```

Comment:
Is this needed for all of `$out/bin/*` or only some, such as `musicxml2ly`? I guess I only (directly) use `lilypond` in practice and haven't hit this issue.

Reply:
Yeah, only for some of the entries.

However, I am not in favor of creating an additional maintenance burden by differentiating between them.